site/content/entry/citra-progress-report-2017-august.md

```diff
@@ -0,0 +1,159 @@
++++
+date = "2017-09-04T15:07:00-04:00"
+title = "Citra Progress Report - 2017 August"
+tags = [ "progress-report" ]
+author = "anodium"
+coauthor = "saphiresurf"
+forum = 0000
++++
+
+2017 has been an amazing year, with more work having been put into the project
+than ever before, but it's not over yet! Last we met was
+[June](/entry/citra-progress-report-2017-june/), and just two months later the
+[Citra issue tracker](https://github.com/citra-emu/citra/pulls) is brimming with
+lots of changes once more. I am extremely excited for this month (and what might
+be of the next month or two) but we're getting ahead of ourselves! On this
+progress report, let's check out the big fish in the July and August pond of patches!
+
+## Updating The Software Renderer ([This](https://github.com/citra-emu/citra/pull/2766), [that](https://github.com/citra-emu/citra/pull/2822), [here](https://github.com/citra-emu/citra/pull/2871), [there](https://github.com/citra-emu/citra/pull/2872), and [those](https://github.com/citra-emu/citra/pull/2891)) by [wwylele](https://github.com/wwylele) and [Subv](https://github.com/Subv)
+
+Citra has two main rendering backends, software and OpenGL, but until very
+recently, no one gave any attention to the software backend, as OpenGL was faster,
```

Comment:
*much attention?
Devs still cared about it, it's very useful for debugging/testing, it just had no fragment lighting.